api/src/test/java/io/kaoto/backend/api/resource/v2/IntegrationsResourceTest.java

```diff
@@ -367,7 +367,7 @@ void beans() throws Exception {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {"Camel Route#route-multi.yaml", "KameletBinding#kamelet-binding-multi.yaml",
```

Comment:
Why remove this KameletBinding? If it is because it is multi, then we should use one that is not multi. But not remove Kamelet Binding from here, shouldn't we?

Reply:
Ah OK, this is the only one KameletBinding here... added the other one.